test/TestBasics.cpp

```diff
@@ -156,6 +161,7 @@ TEST_CASE("Mocxx::Replace() with identical signatures", "[Mocxx]")
   REQUIRE(TrivialPlus(3, 2) == 6);
 }
 
+#ifndef _WIN32
```

Comment:
You don't have `atof` on windows?

Reply:
Windows does have this method and doing a replacement of `atof` does work with the Replace syntax like this:
<image src="https://user-images.githubusercontent.com/3475163/104360172-a903bf00-54de-11eb-9af3-dc6401693691.png" width="75%" />

However in the Replace by name approach it isn't (yet) working:
<image src="https://user-images.githubusercontent.com/3475163/104360236-c2a50680-54de-11eb-9d94-a8564560012f.png" width="75%" />

The only thing I've tried thus far was enabling this disabling of optimization settings: https://docs.microsoft.com/en-us/cpp/build/reference/od-disable-debug?view=msvc-160 (it didn't fix this test).

I do intend on investigating more and hopefully can figure the reason out.

Hope that helps! :smiley: